hack/update-codegen.sh

```diff
@@ -26,3 +26,15 @@ ${SEDNA_ROOT}/hack/generate-groups.sh "deepcopy,client,informer,lister" \
 ${SEDNA_GO_PACKAGE}/pkg/client ${SEDNA_GO_PACKAGE}/pkg/apis \
 "sedna:v1alpha1" \
 --go-header-file ${SEDNA_ROOT}/hack/boilerplate/boilerplate.generatego.txt
+
+# Check if Sedna home is different from the standard directory where GO projects are located
+if [ "${GOPATH}/src/${SEDNA_GO_PACKAGE}/" != "$${SEDNA_ROOT}/" ]; then
+    # Copy generated code into SEDNA_ROOT
+    echo "Copying generated code from ${GOPATH}/src/${SEDNA_GO_PACKAGE}/pkg/ to ${SEDNA_ROOT}/ ..."
+    cp -Rf ${GOPATH}/src/${SEDNA_GO_PACKAGE}/pkg ${SEDNA_ROOT}/
```

Comment:
just move? since we don't expect two same named modules with different location leading confusion?